src/setup.rs

```diff
@@ -114,7 +117,7 @@ fn setup_python_deps(kani_dir: &Path, os: &os_info::Info) -> Result<()> {
     if os.os_type() == os_info::Type::Ubuntu
         && *os.version() == os_info::Version::Semantic(18, 4, 0)
     {
-        crate::os_hacks::setup_python_deps_on_ubuntu_18_04(&pyroot, pkg_versions)?;
+        os_hacks::setup_python_deps_on_ubuntu_18_04(&pyroot, pkg_versions)?;
```

Comment:
Can you move this to be called inside `setup_os_hacks()` instead?

Reply:
Oh. I would expect an `else` case then. From reading this code it does look like do this extra thing for ubuntu 18.4.0.